tests/pdctl/helper.go

```diff
@@ -41,6 +42,16 @@ func ExecuteCommand(root *cobra.Command, args ...string) (output []byte, err err
 	return buf.Bytes(), err
 }
 
+// CreateCommandAndExecuteCommand is used for test purpose when should clean up args.
```

Comment:
Why do we need it?

Reply:
Because the args of command won't initialize at every executing if we use only one `cobra.Command` in a test case.
For example, in `TestPlacementRules` we firstly test with "--group=pd". And if we test with  "--region=1" next, the arg `group` still exists.